api/graphql/schema.graphql

```diff
@@ -527,10 +527,26 @@ input PoolCandidateFilterInput {
     pools: [PoolFilterInput] @builder(method: "App\\Models\\PoolCandidate@filterByPools")
 }
 
+input UserFilterAndOrderInput {
+    pools: [ID] @builder(method: "App\\Models\\User@filterByPools")
+    languageAbility: LanguageAbility @builder(method: "App\\Models\\User@filterByLanguageAbility")
+    expectedClassifications: [ClassificationFilterInput] @builder(method: "App\\Models\\User@filterByClassifications")
+    operationalRequirements: [OperationalRequirement] @builder(method: "App\\Models\\User@filterByOperationalRequirements")
+    locationPreferences: [WorkRegion] @builder(method: "App\\Models\\User@filterByLocationPreferences")
+    hasDiploma: Boolean @scope
+    wouldAcceptTemporary: Boolean @scope
+    jobLookingStatus: JobLookingStatus @builder(method: "App\\Models\\User@filterByJobLookingStatus")
+    isProfileComplete: Boolean @scope
+    skills: [ID] @builder(method: "App\\Models\\User@filterBySkills")
+    isGovEmployee: Boolean @scope
+    orderBy: String   # This needs to be the name of the field in the database eg. first_name
+}
+
 type Query {
     me: User @auth
     user(id: ID! @eq): User @find @guard @can(ability: "view", query: true)
     users: [User]! @all @guard @can(ability: "viewAny")
+    usersPaginated(where: UserFilterAndOrderInput): [User]! @paginate(defaultCount: 10, scopes: ["order"]) @guard @can(ability: "viewAny")
```

Comment:
Should we add a maxCount as well? It could be pretty high. 500? 1000?

Reply:
Sounds good, I'll go with 1000 for now.